benchmark/single-source/NSDictionaryCastToSwift.swift

```diff
@@ -20,6 +20,7 @@ import TestsUtils
 
 @inline(never)
 public func run_NSDictionaryCastToSwift(_ N: Int) {
+#if _runtime(_ObjC)
```

Comment:
In a future pull request, we should update the generation utility to support `#if` defining out entire benchmark functions.